special-pages/pages/new-tab/app/customizer/components/Customizer.module.css

```diff
@@ -56,6 +56,10 @@
             border-color: var(--color-white-at-50);
         }
     }
+
+    &[aria-expanded="true"] {
+        visibility: hidden;
+    }
```

Comment:
This prevents the customizer button from being focusable when the drawer is open